go/vt/vttablet/tabletmanager/rpc_vreplication.go

```diff
@@ -284,8 +494,8 @@ func (tm *TabletManager) UpdateVReplicationWorkflow(ctx context.Context, req *ta
 			tabletTypes = req.TabletTypes
 		}
 		tabletTypesStr := topoproto.MakeStringTypeCSV(tabletTypes)
-		if inorder && req.TabletSelectionPreference == tabletmanagerdatapb.TabletSelectionPreference_UNKNOWN ||
-			req.TabletSelectionPreference == tabletmanagerdatapb.TabletSelectionPreference_INORDER {
+		if (inorder && req.TabletSelectionPreference == tabletmanagerdatapb.TabletSelectionPreference_UNKNOWN) ||
```

Comment:
Can we use the util function `buildTabletTypesString` here as well?

Reply:
Sure. This code was from before I created that function and I only added parens in this PR. But no reason not to update it.

Reply:
Actually, this logic is different so will leave it as-is.